.github/workflows/ci.yml

```diff
@@ -11,11 +11,9 @@ jobs:
 
     strategy:
       fail-fast: false
-      matrix:
-        distro:  [ "tumbleweed", "leap_latest" ]
 
     container:
-      image: registry.opensuse.org/yast/head/containers_${{matrix.distro}}/yast-ruby
+      image: registry.opensuse.org/yast/sle-15/sp5/containers/yast-ruby
```

Comment:
@lslezak is this change correct? It looks like sle 15 container is missing sudo

Reply:
It is OK, there are wrong (old) some Ruby gems, I'm fixing that...

Reply:
Fixed, now CI is green. :heavy_check_mark: